src/trace/step-function-service.ts

```diff
@@ -41,6 +41,12 @@ export class StepFunctionContextService {
     // always triggered by the same event.
     if (typeof event !== "object") return;
 
+    // Legacy lambda parsing
+    const payload = event.Payload;
+    if (typeof payload === "object") {
+      event = payload;
+    }
```

Comment:
Do we really need to define a new constant for this?
```suggestion
    if (typeof event.Payload === "object") {
      event = event.Payload;
    }
```